util/make_timeline.py

```diff
@@ -97,14 +112,28 @@ def parse_file(args):
 
     entries = []
     started = False
-
+    encounter_sets = []
     with args.file as file:
+        # If searching for encounters, divert and find start/end first
+        if args.search_fights:
+            encounter_sets = find_fights_in_file(file)
+            if args.search_fights[0] < 0:
+                for j in range(0, len(encounter_sets)):
+                    encounter_sets[j] = str(j) + '. ' + (" ").join(encounter_sets[j])
+                return encounter_sets
```

Comment:
If you want to be slick:
```
return [f'{i}. {encounter_name}' for i, encounter_name in enumerate(encounter_sets)]
```

Reply:
That's... huh. I had no idea F-string formatting was a thing. There really *is* a tool for everything in Python. That could prove very useful.